code/obj/machinery/interdictor.dm

```diff
@@ -319,15 +390,28 @@
 	throw_range = 5
 	w_class = W_CLASS_NORMAL
 	flags = FPRINT | TABLEPASS | CONDUCT
-	var/interdist = 3
-	//how far the interdictor constructed with this rod will extend its interdiction field
+	var/interdist = 4 //how far the interdictor constructed with this rod will extend its interdiction field
+	var/power_multiplier = null //if present, influences the efficiency of interdictor operation
```

Comment:
```suggestion
	/// how far the interdictor constructed with this rod will extend its interdiction field
	var/interdist = 4
	/// if present, influences the efficiency of interdictor operation
	var/power_multiplier = null 
```

Reply:
Did differently in [a2c5b2c](https://github.com/goonstation/goonstation/pull/12687/commits/a2c5b2c4369795b21aa94d02747fa63a0b8d19a4).